src/Entity/Speaker.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\HttpFoundation\File\File;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\SpeakerRepository")
+ */
+class Speaker
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @Assert\Length(max=255)
+     * @Assert\NotBlank()
+     * @ORM\Column(type="string", length=255)
+     */
+    private $name;
```

Comment:
Use VO for validation

Reply:
I don't think this is mandatory, name has no special constraints

Reply:
length 255 is a constraint :) Same thing for any of the other fields down there.